test/test_legacy_nn.py

```diff
@@ -632,10 +632,12 @@ def _forward(self, module, input):
     def _backward(self, module, input, output, grad_output):
         return module.backward(input, grad_output)
 
-    def _forward_criterion(self, criterion, input, target):
+    def _forward_criterion(self, criterion, input, target, reduce=True):
+        assert(reduce is True)
```

Comment:
never put parenthesis around an `assert` condition. it's a statement so they're not needed, and it increases the risk of bugs (try `assert(False,)` in your interpreter)